src/pages/SearchPage/SearchPage.js

```diff
@@ -17,25 +17,53 @@ async function handleSearch(subreddit) {
 
 function SearchPage() {
   const [isLoading, setIsLoading] = useState(true);
+  const [dataLoaded, setDataLoaded] = useState(false);
   const [searchResults, setSearchResults] = useState([]);
+  const [heatmapInfo, setHeatMapInfo] = useState([]);
   const { subreddit } = useParams();
 
+  function parseRedditData() {
+    const arrInfo = Array(7).fill(0).map(() => Array(24).fill(0));
+    const results = searchResults.data;
+    for (let i = 0; i < results.length; i += 1) {
+      // Add utc seconds from API to base epoch time
+      const date = new Date(0);
```

Comment:
IMO this would look a bit simpler. But I'll leave it up to you.

```
const createdAtInMs = results[i].created_utc * 1000;
const date = new Date(createdAtInMs)
```


Would make the comment above unnecessary, and the less comments required the better. After all, you need to maintain comments as well. Meaning when the impementation changes the comment needs to be adjusted as well. When you forget that, you have a comment that points you in the wrong direction. And that's very confusing.

Sorry for the detour, just wanted to share that :-)